src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/SetDueDate.java

```diff
@@ -28,12 +28,14 @@
     @Override
     public CaseDetails<CaseData, State> apply(final CaseDetails<CaseData, State> caseDetails) {
 
+        log.info("Setting due date.  Case ID: {}", caseDetails.getId());
         if (!caseDetails.getData().getApplication().isSolicitorApplication()) {
-            log.info("Setting holding due date.  Case ID: {}", caseDetails.getId());
-            caseDetails.getData().setDueDate(LocalDate.now(clock).plusWeeks(holdingPeriodInWeeks).plusDays(1));
-
+            if (caseDetails.getData().getApplicationType().isSole()) {
+                caseDetails.getData().setDueDate(LocalDate.now(clock).plusDays(14));
```

Comment:
better to make 14 days configurable

Reply:
there is currently something called AOS_PACK_DUE_DATE_OFFSET_DAYS which im not sure if its the same thing as that is also 14days

Reply:
better to use a separate field say `SOLE_APPLICATION_DUE_DATE_DAYS`